ExperimentIntegrationTests.xctestplan

```diff
@@ -0,0 +1,80 @@
+{
+  "configurations" : [
+    {
+      "id" : "7507FB87-8AE7-4A01-A172-706A9E6B1C1D",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "testTimeoutsEnabled" : true
+  },
+  "testTargets" : [
```

Comment:
The test plan appears empty for me. Not sure if I have to clean an re-build everything to see it out of the box. What I did to finally see the test suite in the test plan is to import it from this view:
<img width="454" alt="Screenshot 2023-07-26 at 12 05 04" src="https://github.com/mozilla-mobile/firefox-ios/assets/1897507/a6132e0a-938b-4153-a507-e70d1247ddf1">
There is a plus button at the end and you can select your ExperimentIntegrationTests.swift file. 
If it is there for you, then discard my comment.